java/code/src/com/redhat/rhn/frontend/xmlrpc/admin/monitoring/AdminMonitoringHandler.java

```diff
@@ -30,6 +33,28 @@
  */
 public class AdminMonitoringHandler extends BaseHandler {
 
+    private static Map<String, String> messageMap;
+    {
+        messageMap = new HashMap<>();
+        messageMap.put("enable", "enable_again_to_sync_config");
+        messageMap.put("disable", "disable_again_to_sync_config");
+        messageMap.put("restart", "restart_needed");
+    }
+
+    private static String toString(boolean enabled, String msg) {
+        String str = enabled ? "enabled" : "disabled";
+        if (StringUtils.isNotEmpty(msg)) {
+            return str + Optional.ofNullable(":" + messageMap.get(msg)).orElse(msg);
```

Comment:
I know I should only look on the testsuite, but I have a feeling that here it will return "enabled:null" or "disabled:null" in case that msg doesn't exist as key in the map, instead of going to the orElse(msg). Take a look :)